cloud-init/add-https-certificate-block-to-cs-profiles.tpl.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Generate a random value for the password.
+password=$(openssl rand -base64 32)
+
+# Convert the full chain and private key PEMs into a single P12 file.
+#
+# This is a Terraform template file, and the domain, full_chain_pem,
+# and priv_key_pem variables are passed in via the templatefile().
+#
+# shellcheck disable=SC2154
+openssl pkcs12 -export -in "${full_chain_pem}" -inkey "${priv_key_pem}" \
+  -out "/tmp/${domain}.p12" -name "${domain}" -passout pass:"$password"
+
+# Create the Java keystore from the P12 file.
+#
+# This is a Terraform template file, and the c2_profile_location and
+# domain variables are passed in via the templatefile().
+#
+# shellcheck disable=SC2154
+keytool -importkeystore \
+  -deststorepass "$password" -destkeypass "$password" \
+  -destkeystore "${c2_profile_location}/${domain}.store" \
+  -srckeystore "/tmp/${domain}.p12" -srcstoretype PKCS12 \
+  -srcstorepass "$password" -alias "${domain}"
+
+# Append the https-certificate block to the Amazon and OCSP C2
+# profiles.
+cat > /tmp/cert-block.txt << CERT_BLOCK
+
+https-certificate {
+  set keystore "${domain}.store";
+  set password "$password";
+}
+CERT_BLOCK
+
+cat /tmp/cert-block.txt >> "${c2_profile_location}/amazon.profile"
+cat /tmp/cert-block.txt >> "${c2_profile_location}/ocsp.profile"
```

Comment:
I was thinking it would be cool to make the list of Cobalt Strike profiles to be modified an input to this script, which should include adding some error-handling (e.g. if it was fed a profile name that doesn't exist).  I don't think it's really necessary now- just wanted to mention it as a potential future improvement.

Reply:
See #121 and commit 2eef0ad.

Reply:
Solid - thanks for writing that up!